builtin/providers/google/resource_google_project.go

```diff
@@ -210,6 +231,18 @@ func resourceGoogleProjectRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("policy_etag", pol.Etag)
 	d.Set("policy_data", string(polBytes))
 
+	// Read the billing account
+	ba, err := config.clientBilling.Projects.GetBillingInfo("projects/" + pid).Do()
```

Comment:
It looks like it's possible to create projects without a billing account (now), and this line of code is causing any projects which do not have a billing account to return a 403 on read.